make flash fails

I receive this error when issuing make flash
 
% make flash
make: *** No rule to make target `build/build/H.bin', needed by `flash'.  Stop.


thanks,

--- a/Makefile
+++ b/Makefile
@@ -269,8 +269,8 @@ ULIBS = -lm
 RULESPATH = $(CHIBIOS)/os/common/startup/ARMCMx/compilers/GCC
 include $(RULESPATH)/rules.mk
 
-flash: build/build/$(PROJECT).bin
-	dfu-util -d 0483:df11 -a 0 -s 0x08000000:leave -D build/ch.bin
+flash: build/$(PROJECT).bin
+	dfu-util -d 0483:df11 -a 0 -s 0x08000000:leave -D build/$(PROJECT).bin
 
 dfu:
 	-@printf "reset dfu\r" >/dev/cu.usbmodem401

